src/Merlin/MerlinProtocolConverter.re

```diff
@@ -32,3 +33,16 @@ let toModelDiagnostics = (errors: MerlinProtocol.errorResult) => {
 
   List.map(f, errors);
 };
+
+let toModelCompletions = (completions: MerlinProtocol.completionResult) => {
+  let f = (cmp: MerlinProtocol.completionResultItem) => {
+    let ret: Model.Actions.completionItem = {
+      completionKind: CompletionKind.Function,
+      completionLabel: cmp.name,
+      completionDetail: Some(cmp.desc),
+    };
+    ret;
```

Comment:
I've seen this pattern a few times, where a seemingly unnecessary variable is created and then immediately returned. I think I've even removed an instance of it in the menu PR, but I'm starting to wonder if there's some stylistic purpose to this that I don't get?

Reply:
Ah, I was using this just for the type annotation... but I realized there is a more concise syntax recently (trying to break the habit though!). I've also left a few where I had logging in the middle and then forgot to revert back. Wish there was a linter rule that could help me catch it!

I've fixed this instance. Thanks for looking the PR over!

Reply:
> Ah, I was using this just for the type annotation... but I realized there is a more concise syntax recently

To get the record fields into scope you mean? There are lots of ways. In case you don't know all of them:

```rust
let f = x => M.{ foo: x, bar: 42 }

let f = x => { M.foo: "", bar: 42 }

let f = x => {
  open M;
  { foo: "", bar: 42 }
}

let f = x => ({ foo: "", bar: 42 }: M.t)

let f = (x: string): M.t => { foo: "", bar: 42 }
```

Reply:
I did not know about all these! This is excellent - thanks for the tips / reference 👍 

I need to use this:
```
let f = x => M.{ foo: x, bar: 42 }
```

and this:
```
let f = x => ({ foo: x, bar: 42 }: M.t)
```

more often to avoid those unnecessary variables.